docs/DeveloperGuide.md

```diff
@@ -458,3 +458,25 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+* Job
+  * Edit a job
+  * Locating jobs by name
+  * Screen jobs by a contact
+  * Support job type (part-time, intern, full-time, contract)
+  * Support other job duration attribute (6-month contract, summer vacation internship)
+  * Support job requirement type (mandatory, good to have)
+  * Support job quota
+  * Smart select company if not found when creating job listing (find & return 1st match)
+  * Support screening candidates via more advanced requirement (salary range, number of skills)
+  * Sorting capability of job list
+* Company
+  * Edit a company
+  * Locating company by name
+  * Enhance company ali
```

Comment:
What does ali mean?

Reply:
Opps, it was supposed to be about company liaise, but I am not sure if I should include it.

Reply:
oh then you can just include it then. Basically just 'Enhance company liaising with the addition of hiring manager contacts' or something like that.